services/core/src/utils/loggerLoki.ts

```diff
@@ -1,22 +1,26 @@
+import dotenv from "dotenv";
 import { createLogger, transports, format } from "winston";
 import LokiTransport from "winston-loki";
 
+dotenv.config();
+
 const loggerInstance = createLogger();
 
-/* loggerInstance.add(
-  new LokiTransport({
-    host: "http://127.0.0.1:3100",
-    basicAuth: "username:password",
-    json: true,
-    format: format.combine(
-      format((info) => {
-        const MESSAGE = Symbol.for("message");
-        info[MESSAGE as any] = JSON.stringify(info.message);
-        return info;
-      })()
-    ),
-  })
-); */
+if (process.env.GRAFANA_LOKI_EXTERNAL_PORT) {
+  loggerInstance.add(
+    new LokiTransport({
+      host: `http://127.0.0.1:${process.env.GRAFANA_LOKI_EXTERNAL_PORT}`,
```

Comment:
Here I'm exposing the loki port in the host, so I'm accessing it with localhost. If I put the docker host then I have to specify another environment variable. In this way when the devs works on the project it doesn't have to do anything. If you think it's better to specify another env variable, it's ok for me

Reply:
Yes, but your "host" when developing was the PC but on staging your "host" will be the container no? 

We need the external port anyway to access over HTTP, so better with a single env var. But my concern is this will not work in the full containerized setup.

> Were you running the server and the UI in containers when you were developing?

I guess you were not?

Reply:
I were not, sorry I dind't reply to that :D